app/services/article_api_index_service.rb

```diff
@@ -81,6 +82,8 @@ def state_articles(state)
     elsif state == "rising"
       Article.published.
         where("positive_reactions_count > ? AND positive_reactions_count < ? AND featured_number > ?", 19, 33, 3.days.ago.to_i)
+    elsif state == "all"
+      base_articles
```

Comment:
This changes the API, I think we should stick with the current behavior, even though could be counter intuitive right now.

I would remove this